pkg/apis/options/legacy_options.go

```diff
@@ -369,32 +369,20 @@ func getPassAccessTokenHeader() Header {
 	}
 }
 
-func getAuthorizationHeader() []Header {
-	headers := []Header{
-		{
-			Name: "Authorization",
-			Values: []HeaderValue{
-				{
-					ClaimSource: &ClaimSource{
-						Claim:  "id_token",
-						Prefix: "Bearer ",
-					},
-				},
-			},
-		},
-		{
-			Name: "x-auth-request-id-token",
-			Values: []HeaderValue{
-				{
-					ClaimSource: &ClaimSource{
-						Claim: "id_token",
-					},
-				},
-			},
-		},
-	}
-	return headers
-}
+// PICS: changed to PicsGetAuthorizationHeader, this one is not used anywhere
+// func getAuthorizationHeader() Header {
+// 	return Header{
+// 		Name: "Authorization",
+// 		Values: []HeaderValue{
+// 			{
+// 				ClaimSource: &ClaimSource{
+// 					Claim:  "id_token",
+// 					Prefix: "Bearer ",
+// 				},
+// 			},
+// 		},
+// 	}
+// }
```

Comment:
Checked with @evozniak, these are tests for our changes, but the tests are failling for quite some time now. We could have a story for fixing/improving coverage in OAuth custom changes. I left the comments because it will be forgotten otherwise.